### ISSUE TITLE
[30k]: Delivery dashboard

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#65]
### Key links

- [Deliverable specification](https://simpler-grants.gitbook.io/wiki/product/deliverables/specifications/delivery-dashboard)
- [Simpler Grants.gov full roadmap](https://github.com/orgs/HHS/projects/12)
- [10k milestones](https://github.com/orgs/HHS/projects/13/views/11?sliceBy%5Bvalue%5D=Delivery+dashboard)

### Description

- **What:** Release a basic public dashboard that allows internal and external stakeholders to track key sprint and delivery metrics for the project.
- **Why:** Publishing our metrics publicly encourages us to define and track key measures for each deliverable and builds public trust in the approach we’re taking. Tracking metrics around sprint and delivery, in particular, will help us scope and plan future deliverables more accurately by enabling us to understand team capacity and delivery velocity.
- **Who:**
  - Project maintainers who want to monitor team capacity and delivery velocity
  - Internal and external stakeholders who want to monitor our progress toward key deliverables

> [!NOTE]
> For more information about this deliverable, please refer to the deliverable specification linked above.

**Comment by EOKENAVA** [comment]
Weekly Update

What has been done?

Met with Billy and team to discuss questions around how to get from 0-1 and what features will bring the most value

Status
Planning

Risk
Curtis still needs to assign a data engineer to this task with the appropriate skillset.

Deliverable Date
TBD

What will be done?

- Curtis to assign a personnel to work on task
- Will investigate the feasibility and timeline for adding the sprint and delivery metrics to the Looker dashboard he shared
- Nava will will propose changes to the [delivery dashboard spec](https://app.gitbook.com/o/cFcvhi6d0nlLyH2VzVgn/s/Pm7UEzeiS1tbLCV1SFRu/product/deliverables/specifications/delivery-dashboard) based on Aaron's findings
- Billy will work on refining and moving [this brain dump about the vision](https://docs.google.com/document/d/1RqBb49B2GmWVy4Pt-gbgmbBgaOLhOamoFxDx2YNZFOI/edit) for [Simpler.Grants.gov](http://simpler.grants.gov/) analytics dashboard into GitBook

**Comment by coilysiren** [comment]
I'm going to try to scope some discovery work where I deploy AWS Redshift and AWS Quicksight

**Comment by coilysiren** [comment]
https://github.com/HHS/simpler-grants-gov/issues/1411
https://github.com/HHS/simpler-grants-gov/issues/1410

**Comment by EOKENAVA** [comment]
**Weekly Update**

**What has been done?**
- Completed technical Planning for Delivery Dashboard 30K

**Status**
On Track

**Risk**
Curtis still needs to assign a data engineer to this task with the appropriate skillset.

**Deliverable Date**
April 30th

**What will be done?**
- Curtis to assign a personnel to work on task
- Public Dashboard Design Discovery - Create a Figma mockup for a dashboard page on simpler.grants.go

**Comment by sarahknoppA6** [comment]
@EOKENAVA is there any update about the Nava review of the deliverable spec? Last update & discussion earlier this week indicated feedback from Nava would be coming, but we need to know when.

**Comment by acouch** [comment]
We have a suggestion to move the work to two 30ks that has a [high-level endorsement](https://github.com/HHS/simpler-grants-gov/issues/1195#issuecomment-1979617119). The April delivery date is still on track. We need to circle back with @widal001, hopefully next week to hammer out the task details. From there we can update the date if necessary.

**Comment by EOKENAVA** [comment]
**Weekly Update**

**What has been done?**
- Completed technical Planning for Delivery Dashboard 30K
- Discussion about goals and scope

**Status**
On Track

**Risk**
- Curtis still needs to assign a data engineer to this task with the appropriate skillset.

**Deliverable Date**
April 30th

**What will be done?**
- Curtis to assign a personnel to work on task
- Breakdown tasks under 30k
- Public Dashboard Design Discovery - Create a Figma mockup for a dashboard page on simpler.grants.gov

**Comment by sarahknoppA6** [comment]
@widal001 see previous comment a couple of comments up- did the team follow up with you on this?

**Comment by sarahknoppA6** [comment]
@EOKENAVA @margaretspring please phrase the risk as an if/then statement. Also, I believe we'd like to discuss today at ToT if there is anything that can be done/started on the delivery dashboard in the absence of the data engineer. cc @lucasmbrown-usds

**Comment by acouch** [comment]
We will have a decision for the dashboard and storage by the end of the sprint.

We will still need to go through the process of creating and estimating the 10Ks and getting approval.

**Comment by margaretspring** [comment]
**Weekly Update**

**What has been done?**
- Completed technical Planning for Delivery Dashboard 30K
- Discussion about goals and scope
- Presented bisected Delivery Dashboard proposal (Simple.grants.gov reporting that's mre end-user oriented & positions us for later iteration. Public GitBook with reporting that might be less approachable for general public, but end-uer iteration will guide transition other reporting venue). Writing up tickets for implementation
- Data scientist onboarding with MH (https://www.linkedin.com/in/ebuwa-evbuoma-fike/)

**Status**
On Track

**Risk**

**Deliverable Date**
April 30th

**What will be done?**
- Curtis to assign a personnel to work on task - 
- Breakdown tasks under 30k
- Public Dashboard Design Discovery - Create a Figma mockup for a dashboard page on simpler.grants.gov

**Comment by margaretspring** [comment]
**Weekly Update - IN DRAFT**

**What has been done?**
-dfg
-

**Status**


**Risk**


**Deliverable Date**


**What will be done?**
-ret

**Comment by margaretspring** [comment]
**Weekly Update 4/3**

**What has been done?**
- Deliverable Specification/30K reviewed & commented by implementation team
- Planning mostly completed, estimates not yet made

**Status**
Planning/implementation

**Risk**
* If Deliverable Specification/30K uncertainty continues, timeline will be impacted as team juggling known & unknowns to plan 10K & tasks
* April 30 date is close w/ decision delays

**Deliverable Date**
Currently slated for April 30

**What will be done?**
- Nava leads to comment on sections of Deliverable Specification/30K needing clarification/unknown until further into work. 
- Getting Embuwa engaged with tasks supporting work 
- Mapping known 10K/tasks/milestones while working through areas needing clarification 
- Close out ADR for business analytics data storage
- Deploy application to AWS ECR
- CI is modified to fail when there are outstanding linter changes
- Identify LOE for Supporting AWS QuickSight vs Hosted BI Tool
- ADR for business intelligence tool for dashboards

**Comment by widal001** [comment]
Updated the ticket format to reduce discrepancies with the deliverable spec.

**Comment by margaretspring** [comment]
**Weekly Update 4/17**

**What has been done?**


**Status**


**Risk**
  

**Deliverable Date**


**What will be done?**

**Comment by margaretspring** [comment]
**Weekly Update 4/24**

**What has been done?**



**Status**


**Risk**


**Deliverable Date**


**What will be done?**

**Comment by lucasmbrown-usds** [comment]
What does "If further BI tool questions come up" mean? Do you just mean questions about whether the tools work for our security requirements etc? Something like, "If during implementation, we learn things (such as how the selected tool can or can't meet security requirements) that make us need to switch our planned approach to the BI tool, then it could take longer to complete this than expected"?

**Comment by acouch** [comment]
This was referring to the security questions.

**Comment by margaretspring** [comment]
**Weekly Update 5/1**

**What has been done?**


**Status**
In Progress

**Risk**
 

**Deliverable Date**


**What will be done?**

**Comment by acouch** [comment]
**Weekly Update 5/8**

**What has been done?**
- Dashboard stakeholder interviews
- Metabase chosen (ADR updated)
- Metabase app (not connected) deployed to AWS
- ELT work initiated

**Status**
- In Progress

**Risk**

- Blocked on Metabase procurement

**Deliverable Date**
Sprint 25 - Jun 11

**What will be done?**
- ETL setup and deployment to tranfer data from github to the analytics database

**Comment by widal001** [comment]
Replacing our old deliverable tickets with a new ticket format
